src/components/value-renderers/Title.tsx

```diff
@@ -6,17 +6,24 @@ import { ValueRenderersProps } from '.';
 
 interface TitleProps extends ValueRenderersProps {
   value?: string;
+  sorted?: 'asc' | 'desc' | false;
 }
 
 const styles = {
   title: css({
     fontWeight: 'bold',
   }),
 };
-export function Title({ value, ...other }: TitleProps) {
+export function Title({ value, sorted = false, ...other }: TitleProps) {
```

Comment:
It does not really make sense to me that the `sorted` prop is added to a generic `Title` component.

I think you should have another component specifically for rendering table headers. It can reuse the `Title` component if needed.

Reply:
table header added thanks